src/common/latest/helpers/__tests__/api-test.ts

```diff
@@ -73,7 +73,7 @@ describe("fetchWithRetry", () => {
       expect.assertions(2);
       try {
         await fetchWithRetry(MOCKED_ENDPOINT, "/api");
-      } catch (error) {
+      } catch (error: any) {
```

Comment:
typically unknown is preffered over any
```suggestion
      } catch (error: unknown) {
```

Reply:
doesn't hurt to be consistent using `error: unknown`, so i've changed it 👍